cmd/nodemon/nodemon.go

```diff
@@ -141,18 +148,29 @@ func run() error {
 	alerts := analyzer.Start(notifications)
 
 	go func() {
-		err := pubsub.StartPubSubMessagingServer(ctx, nanomsgPubSubURL, alerts)
+		err := pubsub.StartPubMessagingServer(ctx, nanomsgPubSubURL, alerts)
 		if err != nil {
 			log.Printf("failed to start pair messaging service: %v", err)
```

Comment:
Is it necessary? I see pair services for telegram and discord below.

Reply:
Nodemon runs a publisher, discord and telegram are subscribers